projects/plugins/super-cache/wp-cache.php

```diff
@@ -3271,14 +3275,39 @@ function wp_cron_preload_cache() {
 						fclose( $fp );
 					}
 				}
+
+				if (
+					$preload_more_taxonomies === false &&
+					count( $details ) !== 1 &&
+					$details[0] !== ''
+				) {
+					$preload_more_taxonomies = true;
+				}
+			}
+
+			if ( $preload_more_taxonomies === true ) {
+				if ( defined( 'DOING_CRON' ) ) {
+					wp_cache_debug( 'wp_cron_preload_cache: scheduling the next taxonomy preload in 3 seconds.' );
+					wp_schedule_single_event( time() + 3, 'wp_cache_preload_hook' );
+					wp_delete_file( $mutex );
+					return;
+				}
 			}
 		}
 	}
 
+	update_option(
+		'preload_cache_counter',
+		array(
+			'c' => ( $c + 10 ),
+			't' => time(),
+		)
+	);
```

Comment:
It would also be neat to make less code need to understand the difference between `'c'` and `'t'` in the preload option.

Reply:
Good point. I checked the code and I never used "t" anywhere. I added comments above this line explaining what it does, which I hope helps, but the code could be tightened up a bit - when the preload is cancelled in either loop, the code is duplicated. I'll put that in it's own function.

Reply:
Thanks. I think that helps. I would like to move it all into a couple of functions that know how to deal with that data, but we can make that a separate PR. :)